src/java.desktop/share/classes/sun/java2d/Spans.java

```diff
@@ -315,19 +315,11 @@ boolean contains(float pos) {
          * position. The end position is ignored
          * in this ranking.
          */
+        @Override
         public int compareTo(Span otherSpan) {
             float otherStart = otherSpan.getStart();
-            int result;
-
-            if (mStart < otherStart) {
-                result = -1;
-            } else if (mStart > otherStart) {
-                result = 1;
-            } else {
-                result = 0;
-            }
 
-            return result;
+            return Float.compare(mStart, otherStart);
```

Comment:
We don't need the variable any more, do we ?

Reply:
inlined